app/scripts/lib/ens-ipfs/setup.js

```diff
@@ -62,7 +62,7 @@ export default function setupEnsIpfsResolver({
 
     // We cannot show this if useAddressBarEnsResolution is off...
     if (useAddressBarEnsResolution && ipfsGateway) {
-      browser.tabs.update(tabId, { url: 'loading.html' });
+      await browser.tabs.update(tabId, { url: 'loading.html' });
```

Comment:
Hmm, should we be `await`-ing here? This might significantly slow down this feature. I don't think it's critical that this process has finished before we proceed with ENS resolution either.

Reply:
thank you for pointing this. So I've tried to remove it, and when I did this the ENS test flakied [once](https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/92007/workflows/5b8a0938-ca15-4ef3-8c78-7eac33162ec5/jobs/3422949/tests#failed-test-0) :thinking: 
An alternative could be to add the `await` only if we are in test. What do you think?

Reply:
Interesting. OK, counterpoint to what I said originally here, this shouldn't take too long to resolve and speed isn't really of the essence here. if the loading screen is shown for too short a time, that might actually be bad because it would look janky. So maybe the `await` is better anyway.

I am curious as to how this resulted in test failure though. It looks like the driver somehow became disconnected from the page.

Reply:
thank you for this info Mark! I'm a bit unsure of the error too, but the jobs have been re-run with the new commits and it hasn't happen again - maybe was un-related to the test/changes.